docs/src/extend/custom-rule-tutorial.md

```diff
@@ -217,7 +217,7 @@ const fooBarRule = require("./enforce-foo-bar");
 const ruleTester = new RuleTester({
   // Must use at least ecmaVersion 2015 because
   // that's when `const` variables were introduced.
-  parserOptions: { ecmaVersion: 2015 }
+  languageOptions: { ecmaVersion: 2015 }
```

Comment:
We should update the same in the example project that is linked from the tutorial:

https://github.com/eslint/eslint/blob/b4d2512809a1b28466ad1ce5af9d01c181b9bf9e/docs/_examples/custom-rule-tutorial-code/enforce-foo-bar.test.js#L10-L14

Reply:
We should probably also update all references to ESLint v8 (e.g, devDependencies, peerDependencies) to ESLint v9.

Reply:
005ee25f29998c8012d1ed95b62f04b85da4d543